packages/flutter_tools/lib/src/test/font_config_manager.dart

```diff
@@ -34,7 +34,11 @@ class FontConfigManager {
   Future<void> dispose() async {
     if (_fontsDirectory != null) {
       globals.printTrace('Deleting ${_fontsDirectory!.path}...');
-      await _fontsDirectory!.delete(recursive: true);
+      try {
+        await _fontsDirectory!.delete(recursive: true);
+      } on Exception {
```

Comment:
nit. be as specific as possible when catching errors (the goal is not to minimize total crash reports, but to avoid those that are not actionable by us).

```suggestion
      } on FileSystemException {
```